### DISCUSSION TITLE
Hey there 👋  Introduce yourself

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#341] [General]
I'll go first. I'm Tabacitu - I'm the creator of Backpack. I've first learnt web dev when I was a teenager, because I saw a friend create a web page that had ANIMATION and I was like "_WHAT?! YOU CAN DO THAT?!_" Unfortunately... there was no animation in the year 2000. But hey... he got me into web dev 😀 And I'm super-grateful for that. 

Over the years, I've freelanced for clients all around the world, I've joined a small web dev company where I helped grow the business, and it helped me grow to a senior dev and then managing partner ([UPDIVISION](https://updivision.com/)) but after I was able to say "_I got this_" about client work... I wanted something more difficult... I wanted _a product_. In the meantime, I've started 7+ other businesses, 6 of which failed 👀 But I learnt so much out of each one of them.

The one business that DIDN'T fail wasn't even a business at all... it was the v3 of an admin panel software I had built, that I open-sourced... and we called it... 🥁🥁🥁 Dick. What, did you expect something else? 😅 Yup, we called it _Dick_, because it was a git repo, and I loved that you had to add it as a remote and do `git pull dick` and `git push dick`... I know, super-mature 😅 Still...

![were-the-millers-no-ragrets-gif](https://user-images.githubusercontent.com/1032474/209810838-7d0c9ffd-2d5d-4524-80ca-e0dcfe84fa49.gif)

I had so much fun creating and maintaining Dick. And writing [the copy for the website](https://web.archive.org/web/20151009010957/http://usedick.com/)... omg was that fun! And apparently... other people liked Dick too, _once they tried it_ 🤷‍♂️😅 But they didn't really like _the name_ - they were afraid their clients would see it in the codebase. 

So in 2016 (I think) I re-wrote the damn thing from scratch, and launched it as Backpack (_A collection of **back**end **pack**ages... get it?!_). Then in time, as more and more people started helping out, and it started taking up 12+ hours of my day... after a few burnouts... I decided to see if it can make some money... if it can maybe pay the bills, so I can keep working on it in a sustainable way... It took us a few long and difficult years... a lot of trial and error... but at some point if became ramen-profitable... and now it's a _pretty decent_ lifestyle business... Hell, we've just doubled the team size, we're 6 devs in the team now 🤯 Which is bonkers to me! We're not making millions _yet_, we haven't created that _ideal company_ where we have fun and everyone it in makes lots of money... but hopefully, if we follow our plan through... we'll get there soon enough... 😉 

In the meantime... working at Backpack... helping devs just like me... and with devs just like me... it's the most fun I've ever had working, without a doubt. I'm grateful _every day_ that I get to do this, so... thank you so much for using Backpack, being part of our community and everything 🙏 _Thank you, thank you, thank you_ 🙏

Ok now it's your turn. Sorry for the long talk... I do that sometimes.

**Comment by malle-pietje** [comment]
After having worked for large IT companies (including Silicon Graphics, Verizon Business Security Services, and EMC) for many years I decided to start my own business in 2015. My company now delivers software solutions and services focused on the UniFi network platform from Ubiquiti and we have customers across the world.

We currently use Backpack for our custom software projects and are considering migrating our existing software solutions across as well. These are currently built on a custom framework that is derived from Userfrosting.

Our major challenge is that the client devices accessing our software solutions are very diverse and in some cases outdated; we regularly see Windows XP devices running old IE versions. This rules out the use of advanced stacks although the back-end admin interface can be more advanced.

Here’s a link to our website: https//artofwifi.net

We also maintain 2 open-source packages: https://github.com/Art-of-WiFi

**Reply by tabacitu** [reply]
Ugh - seeing Windows XP has got to hurt! 🤦‍♂️😅 I'm glad to tell you in Backpack v6 we plan to keep the current interface too, as a theme. So the same IE support you've had so far, you'll still have it! Thanks for sharing @malle-pietje

**Reply by malle-pietje** [reply]
Our main solution provides captive portal for WiFi networks in a large variety of environments which is why we see this huge variety.

**Reply by malle-pietje** [reply]
**Internet Explorer 6.0** is one of the oldest browsers we see...

**Comment by EG-Mohamed** [comment]
Hello Everyone 👋, I Am Mohamed Said They Call me " Medo " I am from EGYPT 🇪🇬  my start was hilarious and unpredictable I started to learn in 2014 and I was playing a game called Conquer Online I was impressed by their website so I decided to learn out of curiosity to know how the backend is going on, I started 2014 as I said and in 2016 I got my first freelancing project I still remember it  😁
since this I have kept working on [Khamsat](https://khamsat.com/user/egyweb) is one of biggest freelancing platform in MENA
in 2019 I started working with Laravel and to be honest, it's one of the best decisions I've taken, in 2021 I started working remotely for a Saudi company besides my freelancing and in 2022 I found Laravel Backpack and it was amazing, the effort on this package is really amazing and started to contribute and I am happy to be part of this great achievement, last but not least I'd like to thank @tabacitu and @pxpm for all the effort 🎉

**Reply by tabacitu** [reply]
Haha it's weird how we didn't learn Web Dev to make a career out of it, but ended up doing it 🎉 We're so lucky, Medo.

Also... I'm so ignorant... today I learnt [what MENA is](https://en.wikipedia.org/wiki/MENA) 🤦‍♂️ Thanks!

**Comment by o15a3d4l11s2** [comment]
Hello, My name is Dimitar and I was always passionate about computers and software development. Around 2003 I started getting more in depth with technologies and programming languages. One of my first projects was a static html page with live clock written in JavaScript 🤯.

In 2008 I started my first paid job as a developer and since then I have went through many different technologies (even Flash 😅). Then in 2009 I started learning Objective-C and writing apps for the iPhone 3GS. End of 2013 I started teaching Laravel 4 to a small group of students. Then forward to 2017 when I started using Backpack 3 and immediately fell in love with how quick and simple it was to develop an admin area for my projects.

Few years ago I started a small company where we do mainly iOS-based development but also a few large web-based projects with Laravel and Backpack.

I love how Backpack and the community around has evolved during the years and much like Tabacitu I felt that during the last 1 - 1.5 years the Gitter community did not get the best experience despite the time he and Pedro (pxpm) have invested in responding to every single question.

I am looking forward to the next iterations of Backpack and I am very happy to hear everyone's story and how Backpack has helped them create something great!

**Reply by tabacitu** [reply]
A LIVE CLOCK?! OMG that's crazy 😅👏👏👏 If you'd start today with JS, I bet it would be a to-do list, done 10 ways, in 10 different JS frameworks, right? 😅

Don't be so modest about your help on Gitter, Dimitar! I didn't really help on Gitter. Yes Pedro did (a lot), but so did _you_! And we're very very grateful for your help! 🙏

**Comment by activeRobert** [comment]
My name is Rob Kehoe, I run an agency that specializes in Laravel development called Active Logic. We're currently the largest shop of Laravel developers in the USA. Our team loves to contribute to open-source Laravel projects as well as some other things! 

here is a link to our website:
https://activelogic.com

Thanks!!

**Reply by malle-pietje** [reply]
Hi Rob, welcome here.

Not to be unfriendly but apparently you're web site is not internationally focused. This is what I see when I access your web site from the Netherlands, one of the many countries in the EU:
![image](https://user-images.githubusercontent.com/12016131/230625207-8906749b-a00e-4bd7-9eb4-8fa01b090973.png)

If this is unintended, you're welcome for the bug report! 👍

**Reply by activeRobert** [reply]
Thanks for letting me know! To avoid spam, we do narrow access, but we obviously need to adjust firewall rules.

**Reply by malle-pietje** [reply]
No problem!

**Reply by tabacitu** [reply]
Hey Rob! Welcome to the community. "_the largest shop of Laravel developers in the USA_" - wow that's a bold statement. Have you guys shipped anything that uses Backpack yet?

**Reply by activeRobert** [reply]
Depending on the project our team uses various tools, backpack being one of them. It's a great tool!

**Comment by chandanpc** [comment]
My name is Chandan Gupta, and I am a developer currently working at TCS (Tata Consultancy Services). I have a passion for programming and exploring new technologies. With a strong background in software development, I enjoy building innovative solutions and solving complex problems.

Apart from my professional endeavors, I have a keen interest in expanding my knowledge through reading and staying up-to-date with the latest advancements in the tech industry. I find joy in learning new programming languages, frameworks, and tools that enable me to enhance my skills and deliver high-quality software.

When I'm not coding or diving into technical literature, you can find me indulging in my love for movies. I enjoy watching a wide range of genres, from thrilling action flicks to thought-provoking dramas. It serves as a great source of entertainment and inspiration for me.

Another aspect of my life that brings me immense joy is traveling. Exploring new places, experiencing different cultures, and meeting people from diverse backgrounds fuels my sense of adventure and broadens my horizons. Whether it's hiking in nature or exploring bustling cities, I find solace and inspiration in the wonders of the world.

Overall, I am a dedicated developer with a thirst for knowledge, a passion for technology, and a love for both intellectual and recreational pursuits. I strive to continuously grow both personally and professionally, pushing boundaries and making a positive impact through my work.

**Reply by tabacitu** [reply]
Hey Chandan 👋 Nice to meet you. How long have you been using Backpack for?

**Reply by chandanpc** [reply]
Hi tabacitu,I am new to backback.
